doc/api/http.md

```diff
@@ -531,8 +531,10 @@ in the response to be dropped and the socket to be destroyed.
 added: v0.11.14
 -->
 
-If a request has been aborted, this value is the time when the request was
-aborted, in milliseconds since 1 January 1970 00:00:00 UTC.
+* {boolean}
+
+The `request.aborted` property will be `true` if the request has
+been aborted.
```

Comment:
Can you add a `changes:` block in the metadata for this property? (e.g. what we have for `createServer()` further below in the file)